clients/java/src/test/java/io/lakefs/clients/api/model/AuthenticationTokenTest.java

```diff
@@ -48,4 +48,12 @@ public void tokenTest() {
         // TODO: test token
     }
 
+    /**
+     * Test the property 'tokenExpiration'
+     */
+    @Test
+    public void tokenExpirationTest() {
```

Comment:
can you please open a ticket for `AuthenticationTokenTest` tests? we have a lot of TODOs here

Reply:
@idanovo I'm not sure that we currently want to implement the tests for our generated clients